portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,7 @@ function addRandomFact() {
 }
 
 function getDataUsingArrowFunctions() {
```

Comment:
the "UsingArrowFunctions" portion is too descriptive about a javascript feature. renaming this to "getData()" should be enough

Reply:
done.